### ISSUE TITLE
Undeclared dependency Import::Into

### ISSUE THREAD

**Issue by eserte** [xsawyerx/anyevent-eris#12]
Tests fail if Import::Into is not installed:

```
Can't locate Import/Into.pm in @INC (you may need to install the Import::Into module) (@INC contains: ...) at t/lib/Eris/Test.pm line 9.
BEGIN failed--compilation aborted at t/lib/Eris/Test.pm line 9.
Compilation failed in require at t/server/debug.t line 1.
BEGIN failed--compilation aborted at t/server/debug.t line 1.
t/server/debug.t ................. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...
```

**Comment by eserte** [comment]
Also undeclared: Net::EmptyPort

**Comment by xsawyerx** [comment]
Finally fixed in 0.000002. Thank you. :)
